autobuild/examples/alldaemons.yaml

```diff
@@ -4,20 +4,24 @@
   daemons:
     # DO NOT DELETE XR_PROXY ENTRY
     - name: XR_PROXY # DO NOT DELETE
-      image: blocknetdximg/exrproxy:0.8.0 # DO NOT DELETE
+      image: blocknetdx/exrproxy:0.8.0 # DO NOT DELETE
       config_mount_dir: /snode/xr_proxy/config # DO NOT DELETE
       nginx_mount_dir: /snode/xr_proxy/nginx # DO NOT DELETE
-    # DO NOT DELETE ETH ENTRY
-    - name: ETH # DO NOT DELETE - Requires 9+ TB NVMe SSD storage (4 Nov, 2021), 16 GB RAM when GETH deployed locally
-      # Only deployed when --deploy_eth or --gethexternal param passed to app.py; Otherwise, ETH entry is ignored.
-      image: blocknetdx/eth-payment-processor:0.5.2 # DO NOT DELETE
-      postgresql_data_mount_dir: /snode/eth_pymt_db # DO NOT DELETE
-      geth_data_mount_dir: /snode # DO NOT DELETE
     # DO NOT DELETE SNODE ENTRY
     - name: SNODE # DO NOT DELETE
       image: blocknetdx/servicenode:latest # DO NOT DELETE
       config_mount_dir: /snode # DO NOT DELETE
       data_mount_dir: /snode # DO NOT DELETE
+    - name: PAYMENT
+      payment_tier1: 35
+      payment_tier2: 200
+      image: blocknetdx/eth-payment-processor:0.5.2 
+      postgresql_data_mount_dir: /snode/eth_pymt_db
+      # DO NOT DELETE ETH ENTRY
+    - name: ETH
+      image: blocknetdx/eth-payment-processor:0.5.2
+      postgresql_data_mount_dir: /snode/eth_pymt_db
+      geth_data_mount_dir: /snode
```

Comment:
These lines should be updated to match the ETH entry currently in `alldaemons.yaml`. We especially don't want to lose the comments currently attached to the ETH entry.

Reply:
feel free to add them.

Reply:
Actually, I don't think I need to add those comments around the ETH entry. I think this section of `alldaemons.yaml` will show up as a conflict when we try to merge this PR. At that time, I *hope* whoever resolves the conflict will resolve the conflicts in favor of the current `alldaemons.yaml` in the area round the ETH entry. (Hopefully it will be me who resolves the conflicts. 🙏.)